PKGBUILD

```diff
@@ -1,28 +1,33 @@
 # Maintainer: Thomas Lübking <thomas.luebking at gmail>
 
 pkgname=qarma
-pkgver=0.1.0
+pkgver=r71.968836b
 pkgrel=1
-pkgdesc="A drop-in replacement clone for zenity, written in Qt4/5"
+pkgdesc="A drop-in replacement clone for zenity, written in Qt5/6"
 arch=('i686' 'x86_64')
 url="https://github.com/luebking/qarma"
 license=('GPL')
-
 depends=('qt5-base')
-makedepends=('gcc')
-license=('GPL')
+makedepends=('git' 'gcc')
+provides=('zenity')
```

Comment:
I'd like to keep the zenity replacement in the extra package as an option so that users/developers can have installed both at the same time to cross-check behavior.

Reply:
done